service/k8sclient/k8sclient.go

```diff
@@ -346,6 +351,34 @@ func init() {
 	}
 
 	pmmClientImage = "percona/pmm-client:" + v.Core().String()
+
+	exists, err := imageExists(context.Background(), pmmClientImage)
+
+	// if !exists or there was an error while checking if the image exists, use dev-latest as default.
+	if !exists || err != nil {
```

Comment:
This seems pretty dangerous.
If there's intermittent issues with connection to the Docker registry, we will default to `dev-latest`.
I suppose this will be the case in production as well, correct?

What about defaulting to `pmmClientImageEnv` instead?
Since the ticket is meant to resolve issues during testing, I think this is a safer approach.